main.tex

```diff
@@ -140,14 +140,28 @@ \subsection{Active hyperlinks}
 a page break (often the error message does not identify this as the underlying problem). In such cases, then simplest solution is to find the
 offending link and enclose it within \verb|\mbox{}|. Note that the {\tt hypperref} package has a `draft' option that will disable all linking; this can be useful to get a draft paper to compile when La\TeX\ has split such a link. This is a better solution while a paper is in development, since the offending link may no longer fall on a page break once all material has been added to the paper.
 
-\section{How to Refer to DESC and LSST} 
+\section{How to Refer to DESC, LSST, the Rubin Observatory, the Telescope, and Camera} 
```

Comment:
```suggestion
\section{How to Refer to DESC, LSST, Rubin Observatory, the Telescope, and Camera} 
```
(the renaming guide says "Neither Vera C. Rubin Observatory nor Rubin Observatory is preceded by “the,” unless there is an object after the name")